src/pages/dmt5/_interest_free_trading.tsx

```diff
@@ -179,4 +183,6 @@ const InterestFreeTrading = () => {
     )
 }
 
+InterestFreeTrading.propTypes = {}
+
```

Comment:
No need for this. If the eslint is firing an error, add a this

```
/* eslint-disable react/prop-types */
```

at the top of the file, till we fix the proper ts config.

Reply:
Are you still experiencing this on your side too @prince-deriv ? Yashim and myself don't seem to get this error when we remove the propTypes and the import of the package on the file itself when converting.

Reply:
@sean-binary this is still occurring to me in some cases. The strange thing is it's not consistent.